errors/errcode.go

```diff
@@ -8,13 +8,24 @@ type ErrCoder interface {
 	GetErrCode() ErrCode
 }
 
-type ErrCode int16
+type ErrCode int32
 
 const (
-	ErrNoCode			ErrCode = -2
-	ErrNoError                      ErrCode = 0
-	ErrUnknown                      ErrCode = -1
-	ErrDuplicatedTx 		ErrCode = 1
+	ErrNoCode               ErrCode = -2
+	ErrNoError              ErrCode = 0
+	ErrUnknown              ErrCode = -1
+	ErrDuplicatedTx         ErrCode = 1
```

Comment:
Can ErrNoCode, ErrUnknown and ErrDuplicatedTx use the same error code like below?